docs/09-drawings.md

```diff
@@ -0,0 +1,328 @@
+  var playerIdleAnimation = playerIdleSheet.getAnimationForAll(game, 125);
+
+  // add drawing to player as "idle"
+  player.addDrawing('idle', playerIdleAnimation);
+  // add player to game
+  game.add(player);
+});
+```
+
+## Sprite Fonts
+
+### Generating the font sheet
+
+You can use tools such as [Bitmap Font Builder](http://www.lmnopc.com/bitmapfontbuilder/) to
+generate a sprite sheet for you to load into Excalibur.
+
+### Creating a sprite font
+
+Start with an image with a grid containing all the letters you want to support.
+
+Once you load it into Excalibur using a [[Texture]] resource, you can create
+a [[SpriteFont]] using the constructor.
+
+For example, here is a representation of a font sprite sheet for an uppercase alphabet
+with 4 columns and 7 rows:
+
+```
+ABCD
+EFGH
+IJKL
+MNOP
+QRST
+UVWX
+YZ
+```
+
+Each letter is 30x30 and after Z is a blank one to represent a space.
+
+Then to create the [[SpriteFont]]:
+
+```js
+var game = new ex.Engine();
+var txFont = new ex.Texture('/assets/tx/font.png');
+// load assets
+var loader = new ex.Loader(txFont);
+
+// start game
+game.start(loader).then(function() {
+  // create a font
+  var font = new ex.SpriteFont(txFont, 'ABCDEFGHIJKLMNOPQRSTUVWXYZ ', true, 4, 7, 30, 30);
+  // create a label using this font
+  var label = new ex.Label('Hello World', 0, 0, null, font);
+  // display in-game
+  game.add(label);
+});
+```
+
+If you want to use a lowercase representation in the font, you can pass `false` for `caseInsensitive`
+and the matching will be case-sensitive. In our example, you would need another 7 rows of
+lowercase characters.
+
+### Font colors
+
+When using sprite fonts with a [[Label]], you can set the [[Label.color]] property
+to use different colors.
+
+### Known Issues
+
+**One font per Label**  
+[Issue #172](https://github.com/excaliburjs/Excalibur/issues/172)
+
+If you intend on changing colors or applying opacity effects, you have to use
+a new [[SpriteFont]] instance per [[Label]].
+
+**Using opacity removes other effects**  
+[Issue #148](https://github.com/excaliburjs/Excalibur/issues/148)
+
+If you apply any custom effects to the sprites in a SpriteFont, including trying to
+use [[Label.color]], they will be removed when modifying [[Label.opacity]].
+
+## Tile Maps
+
+Tile maps are made up of [[Cell|Cells]] which can draw [[TileSprite|TileSprites]]. Tile
+maps support multiple layers and work well for building tile-based games such as RPGs,
+adventure games, strategy games, and others. Cells can be [[Cell.solid|solid]] so
+that Actors can't pass through them.
+
+We recommend using the [Tiled map editor](http://www.mapeditor.org/) to build your maps
+and export them to JSON. You can then load them using a [[Resource|Generic Resource]]
+and process them to create your levels. A [[TileMap]] can then be used as part of a
+level or map class that adds enemies and builds game objects from the Tiled map.
+
+### Creating a tile map
```

Comment:
I've used this documentation before to create a custom tile map and it's explaining a little too much all at the same time. Perhaps the TypeScript example should be spread out across these paragraphs as each concept is handled, so that it's easier to follow along what goes where.

Reply:
Yeah I was considering pulling out Tile Maps into a separate document perhaps

Reply:
I think that is a good idea. It's a whole concept by itself.